### PR TITLE
Choim nse10541 10682 v2

### PR DESCRIPTION
**Pull request by choim-sci**
@covesturtevant, 

This has the change request from PR332, 
 https://github.com/NEONScience/NEON-IS-data-processing/pull/332
and adding actions for get changed files and update pipelines.  

Closing PR332.

### CODE REVIEW

File: .github/actions/build-push/action.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        with:
+            images: ${{ env.GCP_REGISTRY }}/${{ env.IMAGE_NAME }}
+            tags: |
+              # type=semver,pattern={{raw}}
+              # type=semver,pattern={{major}}.{{minor}}
+              # type=semver,pattern={{major}}
+              # type=schedule
+              type=sha
+              # type=ref,event=tag
+              type=ref,event=pr
```
Comment (covesturtevant) [inline .github/actions/build-push/action.yml:45]: Please comment this line. Not needed (for now)

File: .github/actions/build-push/action.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        id: build-and-push
+        uses: docker/build-push-action@v5
+        with:
+            context: ./modules
+            file: ./modules/${{ env.MODULE_NAME }}/Dockerfile
+            # push: ${{ github.event_name != 'pull_request' }}
+            push: true
+            tags: |
+              ${{ steps.meta.outputs.tags }}
+              ${{ env.GCP_REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.semver }}
```
Comment (covesturtevant) [inline .github/actions/build-push/action.yml:60]: Comment this line

File: .github/workflows/DEV_group_loader.yml
Comment (covesturtevant) [inline .github/workflows/DEV_group_loader.yml:1]: Please delete. Test only with errored_datums_reader and processed_datums_reader (use new "neon-is-...) convention when creating image registry in GCP.

File: .github/workflows/DEV_location_loader.yml
Comment (covesturtevant) [inline .github/workflows/DEV_location_loader.yml:1]: Please delete. Test only with errored_datums_reader and processed_datums_reader (use new "neon-is-...) convention when creating image registry in GCP.

File: .github/workflows/neon-is-group-loader.yml
Comment (covesturtevant) [inline .github/workflows/neon-is-group-loader.yml:1]: Test only with errored_datums_reader and processed_datums_reader (use new "neon-is-...) convention when creating image registry in GCP.

File: .github/workflows/neon-is-location-loader.yml
Comment (covesturtevant) [inline .github/workflows/neon-is-location-loader.yml:1]: Test only with errored_datums_reader and processed_datums_reader (use new "neon-is-...) convention when creating image registry in GCP.

**Review by covesturtevant — CHANGES_REQUESTED** [review]
See comments. Thanks!

**Comment by choim-sci** [comment]
@covesturtevant, 
The change requests implemented are below: 
1) *loader.ymls deleted;
2) build-push.yml and build-push-update.yml added
3) for build-push-update: "with image-tag" from wf yml and "inputs" from the action commented to show the push with semantic version clearly
4) for build-push: push with semver tag commented from build-push action to show the push with short-sha

**Review by covesturtevant — APPROVED** [review]

**Pull request merged**